src/test/scala/au/com/agiledigital/toolform/model/PortMappingTest.scala

```diff
@@ -0,0 +1,48 @@
+package au.com.agiledigital.toolform.model
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class PortMappingTest extends FlatSpec with Matchers {
```

Comment:
These would be good candidates for property based tests: https://www.scalacheck.org

You have a particularly good "there and back again" option, because any `PortMapping`'s string representation should be validly parsed into a `PortMapping` equal to the original.

Reply:
I added ScalaCheck tests for `String` -> `PortMapping` -> `String` and `PortMapping` -> `String` -> `PortMapping`. I'm sure I could think up some more if I read into the ScalaCheck documentation for other ideas but I think this will be enough for now in the interest of time.
Tests were added in 770cf69